### PR TITLE
apt-file: update examples

### PR DESCRIPTION
**Pull request by rozie**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/apt-file.md [hunk 1]
```diff
@@ -9,12 +9,12 @@
 
 - Search for packages that contain the specified file or path:
 
-`apt-file {{search|find}} {{part/of/filename}}`
+`apt-file {{search|find}} {{part/of/path/to/file}}`
```
Comment (blueskyson) [inline pages/linux/apt-file.md:12]:
```suggestion
`apt-file {{search|find}} {{part_of_path/to/file}}`
```

File: pages/linux/apt-file.md [hunk 2]
```diff
@@ -9,12 +9,12 @@
...
 
 - List the contents of a specific package:
 
 `apt-file {{show|list}} {{package_name}}`
 
-- Search for packages that match the regular expression given in `pattern`:
+- Search for packages that match the `regular_expression`:
```
Comment (navarroaxel) [inline pages/linux/apt-file.md:18]:
```suggestion
- Search for packages that match a regular expression:
```

File: pages/linux/apt-file.md (hunk 1 above)
Comment (navarroaxel) [reply pages/linux/apt-file.md:12]: What about 
```suggestion
`apt-file {{search|find}} {{partial_path/to/file}}`
```

**Comment by navarroaxel** [comment]
@rozie could you please check this PR?

**Review by pixelcmtd — APPROVED** [review]

**Review by EmilySeville7cfg — APPROVED** [review]
No mandatory fixes found, approved.
Thanks for a contribution! 🚀

**Pull request merged**